src/main/java/at/ac/tuwien/kr/alpha/common/atoms/ComparisonAtom.java

```diff
@@ -37,8 +37,7 @@
 import java.util.stream.Collectors;
 
 /**
- * Represents a builtin atom according to the standard.
- * Copyright (c) 2017-2018, the Alpha Team.
+ * Represents a builtin atom according to the standard. Copyright (c) 2017-2018, the Alpha Team.
```

Comment:
If we keep this copyright statement, it should stay in its own line for better readability. We could also drop it, however, since it is redundant to the one in line 2.

Reply:
removed the lower copyright statement